tests/Oryx.Integration.Tests/LocalDockerTests/DatabaseTestsBase.cs

```diff
@@ -8,51 +8,38 @@
 using System.IO;
 using System.Net.Http;
 using System.Threading.Tasks;
-using JetBrains.Annotations;
 using Microsoft.Oryx.Tests.Common;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Microsoft.Oryx.Integration.Tests.LocalDockerTests
 {
-    public class DatabaseTestsBase
+    public abstract class DatabaseTestsBase
     {
         protected readonly Fixtures.DbContainerFixtureBase _dbFixture;
         protected readonly ITestOutputHelper _output;
+        private readonly Random _rand = new Random();
```

Comment:
Should this be `static`? I say this because each test class inherits from this base and each of it will have its own instance and there could be collisions when `_rand.Next(100)` is called.